sound/soc/sof/intel/hda.c

```diff
@@ -717,7 +717,7 @@ int hda_dsp_probe(struct snd_sof_dev *sdev)
 free_ipc_irq:
 	free_irq(sdev->ipc_irq, sdev);
 free_hda_irq:
-	free_irq(sdev->hda->irq, bus);
+	free_irq(sdev->hda->irq, bus); /* FIXME: is it sdev->hda->irq or sdev->pci->irq ?? */
```

Comment:
should be sdev->hda->irq, though they are actually same values here.
Considering when we are calling pci_irq_vector() with nr!=0, there might be offset there.